File: lista1_py/14.media_ponderada.py
#14. Leia 3 notas de um aluno e o peso de cada nota, calcule e escreva a média ponderada.

#Entrada
nota1 = float(input('Informe a 1ª nota: '))
peso1 = float(input('Informe o peso da nota 1: '))
nota2 = float(input('Informe a 2ª nota: '))
peso2 = float(input('Informe o peso da nota 2: '))
nota3 = float(input('Informe a 3ª nota: '))
peso3 = float(input('Informe o peso da nota 3: '))

#Processamento
media_ponderada = ((nota1 * peso1) + (nota2 * peso2) + (nota3 * peso3)) / (peso1 + peso2 + peso3)

#Saída
print(f'\n>>> A média ponderada das 3 notas é {media_ponderada:.1f}.')

File: lista1_js/14.media_ponderada.js
//14. Leia 3 notas de um aluno e o peso de cada nota, calcule e escreva a média ponderada.

import { question } from "readline-sync"

//Entrada
const nota1 = Number(question('Informe a 1ª nota: '))
const peso1 = Number(question('Informe o peso da nota 1: '))
const nota2 = Number(question('Informe a 2ª nota: '))
const peso2 = Number(question('Informe o peso da nota 2: '))
const nota3 = Number(question('Informe a 3ª nota: '))
const peso3 = Number(question('Informe o peso da nota 3: '))

//Processamento
const media_ponderada = ((nota1 * peso1) + (nota2 * peso2) + (nota3 * peso3)) / (peso1 + peso2 + peso3)

//Saída
console.log(`\n>>> A média ponderada das 3 notas é ${media_ponderada.toFixed(1)}.`)